dto/api_output.py

```diff
@@ -85,6 +85,21 @@ class OutputRecommendationList(Schema):
     rating = fields.Int(required=True)
 
 
+class TempSubscriberDataWithUsernameDto(Schema):
```

Comment:
Is there a version "without username"? If not - please do:

```suggestion
class TempSubscriberDataDto(Schema):
```

Reply:
I also think we name all our dtos with capitals: `DTO`

not sure though